pkg/serializer/internal/metrics/iterable_series_test.go

```diff
@@ -66,34 +63,6 @@ func TestIterableSeriesReceiverStopped(t *testing.T) {
 	iterableSeries.Append(&metrics.Serie{Name: "serie3"})
 }
 
-func TestIterableStreamJSONMarshalerAdapter(t *testing.T) {
```

Comment:
Maybe I'm missing something, but why the test is "unused" now? The test subject still seems to be in the code. Or should `NewIterableStreamJSONMarshalerAdapter` be removed too?

Reply:
This is a good question. 
I wrote initially this test as I wanted to check that `IterableStreamJSONMarshalerAdapter` over a `Series` behaves the same as a `IterableSeries`. As `Series` doesn't implement anymore `StreamJSONMarshaler`, this test must be removed.
As `IterableStreamJSONMarshalerAdapter` is still tested indirectly at several locations (for example in `TestSendV1EventsCreateMarshalersBySourceType`), I did not consider writing a new unit test to only test `IterableStreamJSONMarshalerAdapter`.